conf/default/map.conf

```
@@ -100,8 +100,8 @@ speed_mod: 0
 mob_speed_mod: 0
 
 #Allows you to manipulate the constant multiplier in the skill-up rate formulas, having a potent effect on skill-up rates.
-skillup_chance_multiplier: 2.5
-craft_chance_multiplier: 2.6
+skillup_chance_multiplier: 1.0
```

```suggestion
skillup_chance_multiplier: 1.0
craft_chance_multiplier: 1.0
```
Is this the current chance on retail?

---

See the commit comment. Settings should be for adjusting _from_ retail, but I have my personal doubts as to the retail accuracy of what should be the baseline stock retail level. As far as I know every server has this above 1.0

---

See the commit comment. Settings should be for adjusting _from_ retail, but I have my personal doubts as to the retail accuracy of what should be the baseline stock retail level. As far as I know every server has this above 1.0

---

See the commit comment. Settings should be for adjusting _from_ retail, but I have my personal doubts as to the retail accuracy of what should be the baseline stock retail level. As far as I know every server has this above 1.0

---

See the commit comment. Settings should be for adjusting _from_ retail, but I have my personal doubts as to the retail accuracy of what should be the baseline stock retail level. As far as I know every server has this above 1.0

---

See the commit comment. Settings should be for adjusting _from_ retail, but I have my personal doubts as to the retail accuracy of what should be the baseline stock retail level. As far as I know every server has this above 1.0

---

Thanks shithub